src/scripts/main.js

```diff
@@ -1,7 +1,35 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const body = document.querySelector("body")
+  const message = document.createElement("div")
+  message.classList.add('notification')
+  message.classList.add(type);
+
+
+  const titleElement = document.createElement('h2')
+  titleElement.classList.add('title')
+  titleElement.textContent = title;
+  message.appendChild(titleElement)
+
+
+  const descriptionElement = document.createElement('p')
+  descriptionElement.textContent = description;
+  message.appendChild(descriptionElement)
+
+  message.style.top = `${posTop}px`;
+  message.style.right = `${posRight}px`;
+
+  body.appendChild(message)
+
+  setTimeout(() => {
```

Comment:
The notification disappears after 2 seconds. This might not be enough time for the user to read the message, especially if it's long. Consider increasing the timeout, or better yet, make it a parameter of the function so that it can be customized for each notification.